proto/def.bzl

```diff
@@ -24,22 +24,15 @@ load("@io_bazel_rules_go//go/private:rules/prefix.bzl",
 )
 load("@io_bazel_rules_go//proto:compiler.bzl",
     "GoProtoCompiler",
+    "proto_path",
```

Comment:
Probably better import this as a private symbol.

Reply:
It's in a private file an not exposed through def.bzl, I would rather not have multiple names for the same function, just makes it harder to navigate the code.